Tests/scripts/e2e_tests_report.py

```diff
@@ -0,0 +1,27 @@
+
+from pathlib import Path
+from typing import Any
+
+
+from junitparser import TestSuite, JUnitXml
+
+from Tests.scripts.utils import logging_wrapper as logging
+
+
+E2E_BASE_HEADERS = ["Test File Name"]
+
+
+def calculate_e2e_tests_results(e2e_tests_results_files: dict[str, Path]):
```

Comment:
can this be unified with Tests/Marketplace/print_e2e_test_summary.py above?

Reply:
i will unify them, as in `Tests/scripts/e2e_tests_report.py` we have only one func